packages/main/src/FileUploader.ts

```diff
@@ -303,7 +336,14 @@ class FileUploader extends UI5Element implements IFormInputElement {
 	}
 
 	_onChange(e: Event) {
-		const changedFiles = (e.target as HTMLInputElement).files;
+		let changedFiles = (e.target as HTMLInputElement).files;
+
+		if (!this._checkFileSize(changedFiles)) {
+			if (!this.value) {
+				return;
+			}
+			changedFiles = new DataTransfer().files;
```

Comment:
We need a FileList instead of an `Array` object and the `FIleList` object doesn't have a constructor.
This variable is basically used for input field reset.

Reply:
ack.